ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/Reporter.scala

```diff
@@ -102,14 +102,19 @@ object Reporter {
           }
       }
 
-    override def report(results: Vector[LedgerTestSummary]): Unit = {
+    override def report(results: Vector[LedgerTestSummary], testsSeed: String): Unit = {
```

Comment:
Please use a single name to refer to this information. The original `**identifierSuffix` seems to be a sensible choice.

Reply:
fixed